assets/dev/js/editor/container/container.js

```diff
@@ -337,4 +337,89 @@ export default class Container extends ArgsObject {
 	isDesignable() {
 		return elementor.userCan( 'design' ) && this.isEditable();
 	}
+
+	getSetting( name ) {
+		if ( this.getGlobalKey( name ) ) {
+			const globalValue = this.getGlobalValue( name );
+
+			if ( globalValue ) {
+				return globalValue;
+			}
+		}
+
+		const controlValue = this.settings.get( name );
+
+		if ( controlValue ) {
+			return controlValue;
+		}
+
+		return this.getGlobalDefault( name );
+	}
```

Comment:
For some reason, `getSetting( args.control, true )` on global color still returns the color HEX instead of empty string.
Maybe we need to check for `attributes.__globals__`?

![image](https://user-images.githubusercontent.com/32631382/107611456-b7173d80-6c4c-11eb-9ead-570eadd3efc0.png)